### PR TITLE
Added requests/sec(throughput)

### PR DESCRIPTION
**Pull request by flopp999**
https://github.com/grafana/grafana/issues/9630

### CODE REVIEW

File: public/app/core/utils/kbn.ts [hunk 1]
```diff
@@ -948,6 +949,7 @@ kbn.getUnitFormats = function() {
       text: 'throughput',
       submenu: [
         { text: 'ops/sec (ops)', value: 'ops' },
+        { text: 'requets/sec (rps)', value: 'rps' },
```
Comment (bergquist) [inline public/app/core/utils/kbn.ts]: This conflicts with reads/sec

**Review by bergquist — CHANGES_REQUESTED** [review]

File: public/app/core/utils/kbn.ts (hunk 1 above)
Comment (flopp999) [reply public/app/core/utils/kbn.ts]: I have now changed from rps to reqps

**Review by bergquist — APPROVED** [review]

**Comment by bergquist** [comment]
Thank you for contributing!

File: public/app/core/utils/kbn.ts [hunk 2]
```diff
@@ -948,6 +949,7 @@ kbn.getUnitFormats = function() {
       text: 'throughput',
       submenu: [
         { text: 'ops/sec (ops)', value: 'ops' },
+        { text: 'requets/sec (rps)', value: 'reqps' },
```
Comment (mjtrangoni) [inline public/app/core/utils/kbn.ts]: @bergquist This should be `request` instead of `requets`

**Pull request merged**